python/lsst/meas/algorithms/pcaPsfDeterminer.py

```diff
@@ -234,11 +239,11 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
         nEigenComponents = self.config.nEigenComponents # initial version
 
         if self.config.kernelSize >= 15:
-            self.log.log(-1, \
-                "WARNING: NOT scaling kernelSize by stellar quadrupole moment " +
-                "because config.kernelSize=%s >= 15; using config.kernelSize as as the width, instead" \
-                % (self.config.kernelSize,)
-            )
+            self.log.log(-1,
+                         "WARNING: NOT scaling kernelSize by stellar quadrupole moment " +
+                         "because config.kernelSize=%s >= 15; using config.kernelSize as as the width, instead"
+                         % (self.config.kernelSize,)
+                         )
```

Comment:
I thought we wanted the closing paren to be aligned with the block not the open paren.